docs/codebase-notes/sources-reducer.md

```diff
@@ -127,12 +124,6 @@ Updates `state.breakableLines[sourceId] = breakableLines`
 
 - **Dead** - not referenced anywhere
 
-### `"ADD_BREAKPOINT_POSITIONS"`
```

Comment:
:speech_balloon: I like the cleanup of these notes :) this wasn't necessarily intended to stay up to date anyway, but nice that you did this